programs/pkey/rsa_sign.c

```diff
@@ -46,6 +46,8 @@ int main( void )
 
 #include "mbedtls/rsa.h"
 #include "mbedtls/md.h"
+#include "mbedtls/entropy.h"
```

Comment:
Thanks for this pull request. The CI is falling, the main reason being I think that you need to update the dependencies @ lines 35-37. I'd say that you have to add MBEDTLS_CTR_DRBG_C to that list.

Reply:
The dependency added here: https://github.com/ARMmbed/mbedtls/pull/5423/commits/3208d01d9a490849da5040cb51a1a9182757f19d